.vscode/launch.json

```diff
@@ -2,31 +2,32 @@
   // Use IntelliSense to learn about possible attributes.
   // Hover to view descriptions of existing attributes.
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-  "version": "0.2.0",
+  "version": "0.3.0",
   "configurations": [
     {
       "name": "Run Tests",
       "request": "launch",
       "runtimeArgs": ["run", "test"],
       "runtimeExecutable": "npm",
+      "cwd": "${workspaceFolder}",
       "skipFiles": ["<node_internals>/**"],
-      "type": "pwa-node"
+      "type": "node"
```

Comment:
`pwa-node` is no longer necessary as `node` debugger is no longer in preview. See this https://stackoverflow.com/questions/63442436/what-is-the-pwa-node-type-launch-configuration-on-vscode post for details.